src/scripts/main.js

```diff
@@ -1,3 +1,59 @@
 'use strict';
 
-// write code here
+const buttonAppendRow = document.querySelector('.append-row');
+
+buttonAppendRow.addEventListener('click', () => {
+  const tbody = document.querySelector('tbody');
+
+  if (tbody.children.length < 10) {
+    const trFirst = document.querySelector('tr');
+    const tr = document.createElement('tr');
+
+    for (let i = 0; i < trFirst.children.length; i++) { 
```

Comment:
you can just use [cloneNode](https://developer.mozilla.org/en-US/docs/Web/API/Node/cloneNode) to create new row instead of using loop